src/store/global.ts

```diff
@@ -1,11 +1,18 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { RolesType } from '../components/MembersSectionNew/types/MembersSection.type';
 
 interface globalState {
   isLoggedIn: boolean;
+  first_name: string | null;
```

Comment:
- please use camel case for `first_name` as we are doing with other keys in the global sate